### DISCUSSION TITLE
stark是如何对 computation trace 进行证明的？

### DISCUSSION THREAD

**Discussion by qijinzing** [coset-io/zkp-academy#38] [General]
stark是如何对 computation trace 进行证明的？

**Comment by readygo67** [comment]
可以参考tim老师， https://github.com/Antalpha-Labs/zkp-academy/blob/main/FRI%26Stark/stark101/stark101_arithmetization.md中的解释。
![image](https://github.com/user-attachments/assets/9b7b38fc-fd6f-4bdf-8e28-e4a4fddf9456)
